### ISSUE TITLE
Define Sybil concepts

### ISSUE THREAD

**Issue by poupou-web3** [OpenDataforWeb3/Resources#24]
@stefi_says proposition in [Discord](https://discord.com/channels/1037443230993743902/1038206553326616586/1048375499849728130) is to create content for easier onboarding on the Sybil issue.
I think we could start with something similar to MetricsDAO [Gitbook](https://dictionary.metricsdao.xyz/indexhttps://dictionary.metricsdao.xyz/index) Or with a page in our wiki.

Some of the concepts to define are:

- why fight an mitigate sybil is so important ( democracy, governance, public funding) 
- quadratic funding and  matching
- process of creating wallets (?)
- transaction data and where to find it 
- gas fees 
- zksync and bulk transactions 
- price of forgery
- Grants
- Externally Own Address EOA
- FDD
- Passport
- Stamps
- Gitcoin
- Squelched
- Ceramic network
- DIDs

**Comment by Razga** [comment]
Nice work here. I would add that some structure would help

**Comment by baokiddy** [comment]
Potential categorisation of these concepts into different buckets:

Data dictionary

- data tables
- gas fees, etc.

ODC definitions

- Grants
- Externally Own Address EOA
- FDD
- Passport
- Stamps
- Gitcoin
- Squelched

Explainers/Tutorials

- process of creating wallets (?)
- transaction data and where to find it
- Ceramic network

FAQ

- why fight an mitigate sybil is so important ( democracy, governance, public funding)
- quadratic funding and matching
- zksync and bulk transactions
- price of forgery
- DIDs

@stefi-says @epowell101  let me know what you think

**Comment by baokiddy** [comment]
I'll be sharing a doc for the data dictionary shortly

**Comment by sarob** [comment]
Until we establish a better way, I'd follow PouPou and Epowell's example and create a wiki page to develop this research. More contributors will see it that way.

**Comment by stefi-says** [comment]
Tha wiki page can be found here: [FAQ](https://github.com/OpenDataforWeb3/Landscape/wiki/FAQ)

**Comment by sarob** [comment]
> Tha wiki page can be found here: [FAQ](https://github.com/OpenDataforWeb3/Landscape/wiki/FAQ)

Do want the page to a be generic faq or just anti Sybil faq, ie the page name?

**Comment by poupou-web3** [comment]
@stefi-says I created a hackmd to collaborate on the wiki, https://hackmd.io/@8xd7OsNoS964EomtwwFZaw/rJRftKlFs/edit 
You can sign up with GitHub. It's the first time I'm using so I'm not sure it is the best to use but at least I was able to add some comments and propose some changes. I will be adding other definitions in the next days.

**Comment by sarob** [comment]
Will the comments on the hackMD tool get synchronized with a PR or GitHub issue? We might start excluding people that are not using the tool if not. Just a thought. Sean ***@***.*** On Dec 21, 2022, at 05:58, poupou-web3 ***@***.***> wrote:﻿
@stefi-says I created a hackmd to collaborate on the wiki, ***@***.***/rJRftKlFs/edit
You can sign up with GitHub. It's the first time I'm using so I'm not sure it is the best to use but at least I was able to add some comments and propose some changes. I will be adding other definitions in the next days.

**Comment by poupou-web3** [comment]
I saw there is an option to synchronise with GitHub but I think it is a paid option.
I was blocked by the way we can collaborate on the wiki so I proposed that option. It could also be a google doc or open to any other tool. 
During the Dec 13th meeting, I raised that issue and proposed to open one issue for each question but it was pointed out that it would create many issues, however, this is manageable through GitHub Issues, and issues can be labelled, assigned and filtered easily. 
I think it would be nice to have most of the FAQ done before the start of the hackathon in order to facilitate the hackathon onboarding.

**Comment by epowell101** [comment]
Just a quick note - I like hackMD and I was able to sync on a free version. Seems pretty useful that way. You have to dig around in settings to get it syncd - works bidirectionally as well.

Evan

Evan Powell
415 377 9812
Twitter: @epowell101 ( http://www.twitter.com/epowell101 )

**Comment by poupou-web3** [comment]
Sync is only available on markdown files within the repo and not on the wiki. With the free version, pushing from HackMD can not be done more than 20 times.

**Comment by sarob** [comment]
This has been merged during iteration 1. Closing.
